lib/Doctrine/ruleset.xml

```diff
@@ -93,6 +93,8 @@
     <rule ref="SlevomatCodingStandard.ControlStructures.DisallowYodaComparison"/>
     <!-- Forbid weak comparisons -->
     <rule ref="SlevomatCodingStandard.ControlStructures.DisallowEqualOperators"/>
+    <!-- Require usage of early exit instead of using else -->
```

Comment:
@carusogabriel The comment is not true. The sniff does a little more :)

Reply:
Have a suggestion? :smile:

Reply:
https://github.com/slevomat/coding-standard/commit/4dfc4ed2c43a97983079bbb1d6f2a36833aceb1e 😄